datadog/dogstatsd/base.py

```diff
@@ -559,16 +572,15 @@ def _reset_telemetry(self):
 
     def _flush_telemetry(self):
         telemetry_tags = ",".join(self._add_constant_tags(self._client_tags))
```

Comment:
Do you think we could potentially cache that as well, and then cache the resulting string ? So we end up only passing the numbers?

Reply:
Seems very reasonable and from a cursory look, I think we can precompute the stringified tags just like you said. Overall though it has been a bit quite a hit-or-miss for what seems like should improve the performance vs what actually gives better benchmarking results so I'll need to instrument it. For now I'm making tiny changes here-and-there in obvious places and as I get a bit more comfortable I'll see about these tags!